examples/react-template/screens/Column.tsx

```
@@ -64,21 +66,66 @@ export const ColumnScreen = (): JSX.Element => {
+        {[
+          'Forfait mobile',
+          'Forfait 5GO abonnement seulement',
+          'Forfait internet 5g',
+          'Forfait 70GO abonnement seulement',
+        ].map((item, i) => {
+          return (
+            <Column
+              key={i}
+              narrow
+              verticalAlign={Alignable.ALIGNED_CENTER}
+              ref={(el) => el && (refColumn.current[i] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)